platform/saivpp/vpplib/SwitchStateBaseAcl.cpp

```diff
@@ -563,6 +825,30 @@ sai_status_t SwitchStateBase::AclTblConfig(
     ordered_aces.sort(cmp_priority);
     // SWSS_LOG_NOTICE("# aces %u after sort", ordered_aces.size());
 
+    size_t n_entries = 0;
+    size_t n_tunterm_entries  = 0;
+    for (auto ace: ordered_aces) {
+        p_ace = &aces[ace.index];
+        const sai_attribute_t *attr;
+        bool ttf_flag_set = false;
+        for (uint32_t i = 0; i < p_ace->attrs_count; i++) {
+            attr = &p_ace->attrs[i];
+            if(attr->id == SAI_ACL_ENTRY_ATTR_FIELD_TUNNEL_TERMINATED) {
+                if(attr->value.aclfield.data.booldata == true) {
+                    ttf_flag_set=true;
+                }
+                break;
+            }
+        }
+        if(ttf_flag_set) { 
```

Comment:
Where possible, follow formatting convention of the file (eg. add a space after your if)

Reply:
Updated